### ISSUE TITLE
Documentation for backing up / restoring device

### ISSUE THREAD

**Issue by chotaire** [keybase/client#9439]
It is shocking to see that I have found zero documentation on the web on how to backup and restore a Keybase device, taking into account the impact a user may have when losing this data. This really needs to be added to documentation because otherwise people lose trust.

I've made a start for Windows 10 devices, please feel free to comment and use at free will.
[https://wiki.chotaire.net/keybase](Backing up and restoring a Windows Keybase device)

**Comment by joecorcoran** [comment]
Struggling to help a colleague with this too. They are totally locked out of Keybase after having to restore an iPhone.

https://github.com/keybase/client/issues/9339#issuecomment-344968593

**Comment by dabura667** [comment]
Log in to keybase.io

On your own profile page there’s a cog button on the right side.

Click “reset account” or “revoke all keys” or something like that (the text should be all red... I forget what it says exactly)

**Comment by jake-walker** [comment]
I don't think that you can backup devices but I'd would be a **very** useful feature

**Comment by dabura667** [comment]
Also:

Paper Keys are essentially backups.

By creating a paper key, writing it down and storing it in a safe place, you can always create a new device with the paper key that will have access to any of your current device’s data.

**Comment by soullivaneuh** [comment]
@dabura667 Having a doc to keep the locale device data during re-installation might be a good thing to avoid to create a new device with different name on each of them...

I see two local keybase folder:

```
~/.config/Keybase
~/.config/keybase
```

I suppose it's the second one? The first one looks more for the desktop app.

**Comment by chotaire** [comment]
3rd party documentation updated:  https://wiki.chotaire.net/windows10-keybase
